corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -118,6 +124,49 @@ def process_change(self, change):
     def fire_change_processed_event(self, change, context):
         pass
 
+    def _record_checkpoint_in_datadog(self):
+        sequence = self.get_last_checkpoint_sequence()
+        change_feed = self.get_change_feed()
+        if not isinstance(sequence, dict):
+            topics = change_feed.topics
+            assert len(topics) == 1
+            sequence = {topics[0]: int(sequence)}
+
+        for topic, value in sequence.iteritems():
+            datadog_gauge('commcare.change_feed.processed_offsets'.format(topic), value, tags=[
+                'pillow_name:{}'.format(self.get_name()),
+                'topic:{}'.format(topic),
+            ])
+            datadog_counter('commcare.change_feed.topics.counter'.format(topic), tags=[
```

Comment:
Sorry, this name doesn't mean much to me. How about `change_feed.checkpoint`.

Also is it necessary to split this by topic? Could just have it per pillow.